cli/test/smokehouse/test-definitions/offline-sw-broken.js

```diff
@@ -40,32 +36,9 @@ const expectations = {
     requestedUrl: 'http://localhost:10503/offline-ready.html?broken',
     // This page's SW has a `fetch` handler that doesn't provide a 200 response.
     finalDisplayedUrl: 'http://localhost:10503/offline-ready.html?broken',
-    audits: {
-      'installable-manifest': {
-        score: 0,
-        details: {items: {length: 1}},
-        // TODO: 'warn-not-offline-capable' was disabled in m91. Turn back on once
-        // issues are addressed and check is re-enabled: https://crbug.com/1187668#c22
-        // warnings: {length: 1},
-      },
-    },
-  },
-  artifacts: {
-    InstallabilityErrors: {
-      errors: [
-        // Icon errors were consolidated in M118
-        // https://bugs.chromium.org/p/chromium/issues/detail?id=1476999
-        {
-          _minChromiumVersion: '118',
-          errorId: 'no-acceptable-icon',
-        },
-        {
-          _maxChromiumVersion: '117',
-          errorId: 'no-icon-available',
-        },
-      ],
-    },
+    audits: {},
```

Comment:
There aren't really any expectations her anymore. I think we can just remove this smoke test.